DESCRIPTION

```diff
@@ -40,10 +40,12 @@ Imports:
 Suggests: 
     rmarkdown,
     r2dii.plot,
-    testthat (>= 3.0.0)
+    testthat (>= 3.0.0),
+    spelling
 Config/testthat/edition: 3
 Encoding: UTF-8
 LazyData: true
 Roxygen: list(markdown = TRUE)
 RoxygenNote: 7.1.2
 URL: https://2degreesinvesting.github.io/r2dii.climate.stress.test
+Language: en-GB
```

Comment:
US english please, @jacobvjk has a big inclination to the states 😅

Reply:
Done